fudge-ratio/src/main/webapp/script.js

```diff
@@ -71,6 +81,57 @@ buttonClear.addEventListener("click", () => {
   cleanSessionState();
 });
 
+buttonAddHour.addEventListener("click", () => {
+    durationHour += 1;
+    durationText.textContent = padd(durationHour) + ":" + padd(durationMin) + ":" + padd(durationSec);
+});
+
+buttonMinusHour.addEventListener("click", () => {
```

Comment:
I think you all can agree with me we need more files and modules so we don’t end up with a really big file with lots of globals and spaghetti code.

Hoping to merge this PR and the next  so I can create a proper file structure with modules and then use a bundler